docs/src/modules/components/AppFrame.js

```diff
@@ -199,6 +205,15 @@ class AppFrame extends React.Component<any, any> {
                 <LightbulbOutline />
               </IconButton>
             </Tooltip>
+            <Tooltip title="Toggle right-to-left/left-to-right" enterDelay={300}>
+              <IconButton
+                color="contrast"
+                aria-label="change direction"
+                onClick={this.handleToggleDirection}
+              >
+                {rtl ? <FormatTextdirectionLToR /> : <FormatTextdirectionRToL />}
```

Comment:
In the long run, I want to add a dynamic palette section too so people can try out what it looks like with different primary, secondary colors. It's fine this way but it won't scale. We gonna need to find a different solution for the dynamic palette color.

Reply:
@oliviertassinari It's not very hard to implement I've done something similar before. We can keep a raw theme object (one you can pass to createMuiTheme) with an action `SET_THEME` any deep change will cause the recreation of the entire object. lodash has useful utils `get` and `set` for this.

I'd say we make an issue for that. It's not in the same scope as this change.

Reply:
Oh no no, I think there is a misunderstanding. We already have an issue for this. Also, we don't need any lodash function as it's a matter of call the `getMuiTheme` with the right arguments. My point was mostly about the docs UI/UX. You can ignore my comment for this PR.